.circleci/config.yml

```diff
@@ -0,0 +1,99 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/prison-visits-2
+    docker:
+      - image: circleci/ruby:latest-node-browsers
+        environment:
+          RAILS_ENV: test
+          PG_HOST: localhost
+          PG_USER: ubuntu
+          RAILS_ENV: test
+          RACK_ENV: test
+      - image: circleci/postgres:9.4.12-alpine
+        environment:
+          POSTGRES_USER: ubuntu
+          POSTGRES_DB: pvb2_test
+    steps:
+      - checkout
+      - restore_cache:
+          key: prison-visits-2-{{ checksum "Gemfile.lock" }}
+      - run: bundle check --path vendor/bundle || bundle install --path vendor/bundle
+      - run:
+          name: install firefox
+          command: |
+            wget -L "https://ftp.mozilla.org/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2" -O "firefox-$FIREFOX_VERSION.tar.bz2"
+            tar xjf "firefox-$FIREFOX_VERSION.tar.bz2"
+            sudo rm -rf /opt/firefox
+            sudo mv firefox /opt/
+            sudo ln -sf /opt/firefox/firefox /usr/bin/firefox
+      - save_cache:
+          key: prison-visits-2-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+      - run:
+          name: install geckodriver
+          command: |
+            wget https://github.com/mozilla/geckodriver/releases/download/v0.19.0/geckodriver-v0.19.0-linux64.tar.gz
+            tar -zxvf geckodriver-v0.19.0-linux64.tar.gz
+            sudo mv geckodriver /usr/local/bin/
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
+      - run: bundle exec rake
+      - store_test_results:
+          path: coverage
+  # build:
```

Comment:
Can you add a comment about the commented out config? That way someone with no context about it will know a bit what is going on